README.md

```diff
@@ -1,27 +1,148 @@
+  const lineLayer = {
+    type: "line",
+    x: "_time",
+    y: "_value",
+  }
 
-## Thanks
+  const config = {
+    table: table,
+    layers: [lineLayer],
+  }
+  </pre>
+
+3. Render your component by passing the `confg` object as the config prop to the `<Plot>` component. Be sure that the parent component around `<Plot>` has both a height and a width measured in positive values. If either is not a positive value, the graph will not be visible.
+
+   For example, to make a `<Plot>` that adjusts to screen height and width, in your React rendering code return this element:
+
+  <pre>
+  // return this element in your React rendering code:
+
+  &#60;div
+    style={{
+      width: "calc(70vw - 20px)",
+      height: "calc(70vh - 20px)",
+      margin: "40px",
+    }}
+  &#62;
+    &#60;Plot config={config} /&#62;
+  &#60;/div&#62;
+  </pre>
+
+#### Example Using Flux [](#example-using-flux)
+
+When generating the table through a Flux result:
+
+- call the `fromFlux` utility function on the csv that is generated by Flux
+- get the table in the returned object from calling `fromFlux`
+
+Here is an example of turning a result in comma separate values (CSV) from Flux into a table and rendering it without optional properties:
+
+  <pre>
+  import {Plot, fromFlux} from '@influxdata/giraffe'
+
+  // ...
+
+  const fluxResultCSV = `#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,double,string,string,string,string
+#group,false,false,true,true,false,false,true,true,true,true
+#default,_result,,,,,,,,,
+,result,table,_start,_stop,_time,_value,_field,_measurement,example,location
+,,0,2020-03-25T20:58:15.731129Z,2020-04-24T20:58:15.731129Z,2020-04-03T18:31:33.95Z,29.9,value,temperature,index.html,browser
+,,0,2020-03-25T20:58:15.731129Z,2020-04-24T20:58:15.731129Z,2020-04-03T18:55:23.863Z,28.7,value,temperature,index.html,browser
+,,0,2020-03-25T20:58:15.731129Z,2020-04-24T20:58:15.731129Z,2020-04-03T19:50:52.357Z,15,value,temperature,index.html,browser
+,,0,2020-03-25T20:58:15.731129Z,2020-04-24T20:58:15.731129Z,2020-04-03T19:53:37.198Z,24.8,value,temperature,index.html,browser
+,,0,2020-03-25T20:58:15.731129Z,2020-04-24T20:58:15.731129Z,2020-04-03T19:53:53.033Z,23,value,temperature,index.html,browser
+,,0,2020-03-25T20:58:15.731129Z,2020-04-24T20:58:15.731129Z,2020-04-03T20:19:21.88Z,20.1,value,temperature,index.html,browser
+,,0,2020-03-25T20:58:15.731129Z,2020-04-24T20:58:15.731129Z,2020-04-10T22:20:40.776Z,28.7,value,temperature,index.html,browser
+`
+
+  const dataFromFlux = fromFlux(fluxResultCSV)
+
+  const lineLayer = {
+    type: "line",
+    x: "_time",
+    y: "_value",
+  }
+
+  const config = {
+    table: dataFromFlux.table,
+    layers: [lineLayer],
+  }
+
+  // ...
+
+  // return this element in your React rendering code:
+
+  &#60;div
+    style={{
+      width: "calc(70vw - 20px)",
+      height: "calc(70vh - 20px)",
+      margin: "40px",
+    }}
+  &#62;
+    &#60;Plot config={config} /&#62;
+  &#60;/div&#62;
```

Comment:
We can't use actual divs inside a `<pre>` if we want to show the literal "`<div>`" as text. I chose `<pre>` over triple \` marks because I wanted to indent the whole code block, which `<pre>` allows me to do while \` marks do not.